build_tools/circle/list_versions.py

```diff
@@ -32,10 +33,23 @@ def human_readable_data_quantity(quantity, multiple=1024):
             quantity /= multiple
 
 
-def get_pdf_size(version):
+def get_file_extension(version):
+    if 'dev' in version:
+        # The 'dev' branch should be explictly handled
+        return 'zip'
+
+    current_version = LooseVersion(version)
+    min_zip_version = LooseVersion('1.0.0')
```

Comment:
I think we should update:

```suggestion
    min_zip_version = LooseVersion('0.24.1')
```

Reply:
I think that we can avoid this change because we were able to build the PDF on 0.24.0 and in this way we have the same code in 0.24.X and master (or it avoids making this change in master).